### DISCUSSION TITLE
Render replays to video

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [ppy/osu#17971] [Ideas] [answered]
I feel like a tool in lazer to render a play you just made or past replays to mp4 would be really useful for a lot of people, I know there's already third party tools but they're not that great.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/12986. Please search, that's important to the point it got put right in front of the discussion listing page:
<img width="768" alt="CleanShot 2022-04-25 at 06 16 23@2x" src="https://user-images.githubusercontent.com/22781491/165015004-3fefa22c-5b64-4c80-afea-fcf4514e88be.png">
